libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -133,7 +133,8 @@ static_assert(maximum_tracked_dpos_confirmations >= ((max_producers * 2 / 3) + 1
 /**
  * Maximum number of finalizers in the finalizer set
  */
-const static int max_finalizers = max_producers;
+const static size_t max_finalizers = 64*1024;
+const static size_t max_finalizer_description = 256;
```

Comment:
detail: I think I would have named it `finalizer_description_max_size`

Reply:
Added `_size`